SYCL/ESIMD/mandelbrot/mandelbrot.cpp

```diff
@@ -164,7 +164,8 @@ int main(int argc, char *argv[]) {
   fclose(dumpfile);
 
   bool passed = true;
-  if (!esimd_test::cmp_binary_files<unsigned char>(out_file, argv[2], 0)) {
+  if (!esimd_test::cmp_binary_files<unsigned char>(out_file, argv[2], 0,
+                                                   0.0027)) {
```

Comment:
Please define tolerance definition macro for `0.0027` along with comment on why percentage-tolerance is added here.

Reply:
BTW, it also should be for ext_intel_esimd_emulator backend only.

Reply:
Check this

https://github.com/intel/llvm-test-suite/blob/857b056e46a44b925f84974a4145d80437ca9cb9/SYCL/ESIMD/matrix_transpose_glb.cpp#L251

Reply:
Yep, have found it.